src/selectionHandler.ts

```diff
@@ -86,7 +86,7 @@ export function getSelectionHandler(
       void setState(eventData).then(() => {
         void doPollNotifications(context);
         void doPollStatus(context);
-        void setHover(context, position);
+        void setHover(context, editor.selection.active.translate(0,10));
```

Comment:
extract `editor.selection.active.translate(0,10)` to a function, something that'll look like `marginRight(position)`